composer.lock

```diff
@@ -10228,18 +10230,18 @@
     "platform": {
         "php": ">=7.1.3",
         "ext-curl": "*",
+        "ext-dom": "*",
         "ext-fileinfo": "*",
-        "ext-intl": "*",
-        "ext-mbstring": "*",
-        "ext-zip": "*",
+        "ext-gd": "*",
         "ext-iconv": "*",
+        "ext-intl": "*",
         "ext-json": "*",
-        "ext-gd": "*",
-        "ext-dom": "*"
+        "ext-mbstring": "*",
+        "ext-zip": "*"
     },
     "platform-dev": [],
     "platform-overrides": {
         "php": "7.1.3"
     },
-    "plugin-api-version": "2.0.0"
+    "plugin-api-version": "1.1.0"
```

Comment:
I dont think we want this 🤔

Reply:
So we decide to force the usage of composer v2?

Reply:
FYI, it also work with composer v2 
![image](https://user-images.githubusercontent.com/1462701/99076692-d5bd5800-25bb-11eb-9e32-76c84689502e.png)